core/tests/protractor_desktop/topicAndStoryEditorFileUploadFeatures.js

```diff
@@ -52,17 +53,20 @@ describe('Topic editor functionality', function() {
   beforeEach(function() {
     users.login('creator@topicEditor.com');
     topicsAndSkillsDashboardPage.get();
-    topicsAndSkillsDashboardPage.editTopic(topicName);
   });
 
   it('should edit topic name, abbreviated topic name, ' +
     'thumbnail and description correctly', function() {
-    newTopicName = 'Topic 1 edited';
-    topicEditorPage.changeTopicName(newTopicName);
+    topicsAndSkillsDashboardPage.get();
+    topicsAndSkillsDashboardPage.createTopic('TASEFUF_1', 'abbrev');
```

Comment:
Can we also use a constant here? Otherwise `TASEFUF_1` is difficult to understand.

Also, how do you think we should create reliably unique names? We should probably create a standard for this to ensure uniqueness.